library-core/src/main/java/com/mikepenz/fastadapter/FastAdapter.java

```diff
@@ -484,14 +507,14 @@ public void registerTypeInstance(Item item) {
      * @return the Item typeInstance
      */
     public Item getTypeInstance(int type) {
-        return mTypeInstances.get(type);
+        return getTypeInstanceCache().get(type);
     }
 
     /**
      * clears the internal mapper - be sure, to remap everything before going on
      */
     public void clearTypeInstance() {
```

Comment:
This is very critical and maybe should be removed in favor of ```fastAdapter.getTypeInstanceCache().clear();```

Reply:
`getTypeInstanceCache` is anyways just returning the TypeInstanceCache per `FastAdapter` so it'S fine. or do I miss something?